benchmarks/callbacks/benchmark_SequenceOfRegionTimer.cpp

```diff
@@ -0,0 +1,149 @@
+#include <iostream>
+
+#include "benchmark/benchmark.h"
+
+#include "Kokkos_Core.hpp"
+
+#include "kokkos-utils/callbacks/EventNameMatcher.hpp"
+#include "kokkos-utils/callbacks/Manager.hpp"
+#include "kokkos-utils/callbacks/SequenceOfRegionTimer.hpp"
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+namespace Kokkos::utils::benchmarks::callbacks
+{
+struct MyFixture
+{
+    void run(const ::benchmark::State& state) const
+    {
+        Kokkos::Tools::pushRegion("region A");
+        Kokkos::Tools::popRegion();
+
+        Kokkos::Tools::pushRegion("region B");
+        Kokkos::Tools::popRegion();
+
+        //! Add some regions that won't match, it's just noise.
+        for(unsigned short int irep = 0; irep < state.range(0); ++irep) {
```

Comment:
Just noting that "google benchmark" is using `int64_t`:

(checked with `static_assert(std::same_as<decltype(state.range(0)), int64_t>);`)

Reply:
:heavy_check_mark: